### PR TITLE
IKASAN-1734 - first pass of creating a cli distribution zip and using…

### PR DESCRIPTION
**Pull request by kieronedwards**
… in filesystem archetype to review. Notes

- Created a ikasan-simple version of the scripts that dont use the config service
- The batch scripts dont run on my windows machine - have these been tested
- In ikasan.sh assume defaults for config-service, but no clean way to derive the module name (using the directory name is not clean). This could be put in env.sh - but why not use local application.properties ?
- Suggest we should re-consider having a common application.properties in addition to using the config-service (which only has environment specific properties). The module name would be in these properties
- Even if config-service properties not passed in the shell still tries to locate a config-service
- shell unit tests are failing despite no change to codebase :- (- File [.\pid\Module_sampleProcess] not found)
- corrected naming of some poms

### CODE REVIEW

File: ikasaneip/cli/bin/ikasan.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Do not change this script, change the env.sh script fpr you custom environment settings.
+
+SCRIPT_DIR=$(pwd)
+
+LOCAL_ENV=$(pwd)/config-service-env.sh
+if [[ -f "$LOCAL_ENV" ]]; then
+    . ${LOCAL_ENV}
+fi
+
+CONFIG_SERVICE_URL=${CONFIG_SERVICE_URL:-'http://localhost:8880'}
```
Comment (andrzej-majewski) [inline ikasaneip/cli/bin/ikasan.sh]: is this bash notation to defaulting variable

Comment (mitcje) [reply ikasaneip/cli/bin/ikasan.sh]: I would keep the script very simple so its more likely to work across shells

**Review by andrzej-majewski — APPROVED** [review]

**Review by andrzej-majewski — DISMISSED** [review]

File: ikasaneip/cli/shell/bin/ikasan-simple.bat [hunk 2]
```diff
@@ -42,18 +43,6 @@ rem  * ====================================================================
 rem  */
 
 rem Do not change this script, change the env.sh script fpr you custom environment settings.
-
-echo off
+rem this script assumes you are using the filesystem for application.properties
 set currentDir=%CD%
```
Comment (mitcje) [inline ikasaneip/cli/shell/bin/ikasan-simple.bat:47]: Should we not be loading a simple env.bat ?

File: ikasaneip/cli/shell/bin/ikasan-simple.sh [hunk 3]
```diff
@@ -44,19 +44,10 @@
 #!/bin/bash
 #set -u
 
-# Do not change this script, change the env.sh script fpr you custom environment settings.
-
+# Do not change this script, change the env.sh script for you custom environment settings.
+# this script assumes you are using the filesystem for application.properties
```
Comment (mitcje) [inline ikasaneip/cli/shell/bin/ikasan-simple.sh:48]: Should we not be loading a simple env.sh ?

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-filesystem-im/src/main/resources/archetype-resources/bin/env.sh [hunk 4]
```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
```
Comment (mitcje) [inline ikasaneip/developer/mvn-archetype/ikasan-standalone-filesystem-im/src/main/resources/archetype-resources/bin/env.sh:1]: Why are we deleting this?

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-filesystem-im/src/main/resources/archetype-resources/bin/env.bat [hunk 5]
```diff
@@ -1 +0,0 @@
-echo "No additional environment settings."
```
Comment (mitcje) [inline ikasaneip/developer/mvn-archetype/ikasan-standalone-filesystem-im/src/main/resources/archetype-resources/bin/env.bat:1]: Why are we deleting this?

**Review by mitcje — CHANGES_REQUESTED** [review]

**Review by mitcje — APPROVED** [review]

**Pull request merged**